packages/shared/src/events.ts

```diff
@@ -0,0 +1,106 @@
+import { Logger, ManageLogger } from './types';
```

Comment:
To me it seems a little inconsistent that we have `types` which contains all our types except the event-related types which are here.

I think we should be consistent and either keep everything in a single types file, or create a `types/` dir with a few different files in it, one of which would be this one.

If we want to go the second route, I suggest we create a refactor ticket and we keep this the way it is in this PR. If you want to put everything in the one types file, please do that in this PR.

What do you think @lukas-reining ?

Reply:
I would go for the types /dir and do the refactor ticket.

Reply:
I will create the ticket!

Reply:
Created: https://github.com/open-feature/js-sdk/issues/437